tests/defender/system/aws_iot_tests_defender_system.c

```diff
@@ -911,17 +916,34 @@ static void _verifyTcpConections( int total,
 static char * _getIotAddress( void )
```

Comment:
This function should be re-implemented using the Defender metrics function to keep it independent of the network stack.

Reply:
Note: Currently in freertos repository, it uses secure sockets API, which will also need change when we want to go sub-module route.

Reply:
Removed the test of verifying server address as discussed.